packages/portalnetwork/src/client/provider.ts

```diff
@@ -34,7 +35,19 @@ export class UltralightProvider {
   }
 
   static async create(opts: Partial<PortalNetworkOpts>): Promise<UltralightProvider> {
-    const portal = await PortalNetwork.create(opts)
+    const finalOpts = {
+      ...DEFAULT_OPTS,
+      ...opts,
+    }
+    
+    if (!finalOpts.bindAddress) {
```

Comment:
Is this check really needed since you're providing a default value in `DEFAULT_OPTS`?